modern/README.md

```diff
@@ -0,0 +1,193 @@
+These images are mainly focused for Conan Center CI.
+
+#### GCC
+
+| Version                                                                                                   | Arch    |  Status, Life cycle  |
+|-----------------------------------------------------------------------------------------------------------|---------|----------------------|
+| [conanio/gcc5-ubuntu16.04-jenkins: gcc 5](https://hub.docker.com/r/conanio/gcc5-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc6-ubuntu16.04-jenkins: gcc 6](https://hub.docker.com/r/conanio/gcc6-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc7-ubuntu16.04-jenkins: gcc 7](https://hub.docker.com/r/conanio/gcc7-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc8-ubuntu16.04-jenkins: gcc 8](https://hub.docker.com/r/conanio/gcc8-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc9-ubuntu16.04-jenkins: gcc 9](https://hub.docker.com/r/conanio/gcc9-ubuntu16.04-jenkins/)     | x86_64  |  Supported           |
+| [conanio/gcc10-ubuntu16.04-jenkins: gcc 10](https://hub.docker.com/r/conanio/gcc10-ubuntu16.04-jenkins/)  | x86_64  |  Supported           |
+| [conanio/gcc11-ubuntu16.04-jenkins: gcc 11](https://hub.docker.com/r/conanio/gcc11-ubuntu16.04-jenkins/)  | x86_64  |  Supported           |
+
+
+#### Clang
+
+| Version                                                                                                                  | Arch   |  Status, Life cycle  |
+|--------------------------------------------------------------------------------------------------------------------------|--------|------------|
+| - [conanio/clang8-ubuntu16.04-jenkins: clang 10](https://hub.docker.com/r/conanio/clang10-ubuntu16.04-jenkins/)          | x86_64 |  Supported |
+| - [conanio/clang8-ubuntu16.04-jenkins: clang 11](https://hub.docker.com/r/conanio/clang11-ubuntu16.04-jenkins/)          | x86_64 |  Supported |
+| - [conanio/clang8-ubuntu16.04-jenkins: clang 12](https://hub.docker.com/r/conanio/clang12-ubuntu16.04-jenkins/)          | x86_64 |  Supported |
+
+
+
+Use the images locally
+======================
+
+You can also use the images locally to build or test packages, this is an example command:
+
+```
+docker run --rm -v /tmp/.conan:/home/conan/.conan conanio/gcc63 bash -c "conan install boost/1.74.0@ --build"
+```
+
+This command is sharing ``/tmp/.conan`` as a shared folder with the conan home, so the Boost package will be built there.
+You can change the directory or execute any other command that works for your needs.
+
+If you are familiarized with Docker compose, also it's possible to start a new container by:
+
+```
+docker-compose run -v /tmp/.conan:/home/conan/.conan gcc11 bash -c "conan install boost/1.74.0@ --build"
```

Comment:
I was asking because of the `gcc11`, only that single set of characters there. If using an image name it works, sure... but, what is `gcc11` referring to?

Reply:
Oh! I didn't notice the `docker-compose` file. Thanks!